Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js

```diff
@@ -45,6 +45,19 @@ type Props = $ReadOnly<{|
    */
   keyboardDismissMode?: ?('none' | 'on-drag'),
 
+  /**
+   * Specifies the background color of the drawer. The default value is white.
+   * If you want to set the opacity of the drawer, use rgba. Example:
+   *
+   * ```
+   * return (
+   *   <DrawerLayoutAndroid drawerBackgroundColor="rgba(0,0,0,0.5)">
+   *   </DrawerLayoutAndroid>
+   * );
+   * ```
+   */
+  drawerBackgroundColor?: ColorValue,
```

Comment:
@lunaleaps adding drawerBackgroundColor as an optional Prop here.

Reply:
Nit: Let's make this `drawerBackgroundColor?: ?ColorValue` in case people spread an object with `drawerBackgroundColor: null`

Reply:
sure @lunaleaps  will push a commit with that change so it covers all edge cases!